cmd/snap-repair/cmd_run.go

```diff
@@ -99,6 +101,14 @@ func (c *cmdRun) Execute(args []string) error {
 				// no more repairs
 				break
 			}
+
+			// if the store is offline, we want the unit to succeed and not
+			// report failures
+			if errors.Is(err, errStoreOffline) {
+				logger.Debugf("running snap repair: %v", err)
```

Comment:
On the one hand I'd use something like `logger.Notice("store access is disabled")`, as it'd be nice to at least log that that we're in such state, just in case someone tries to find out why repairs aren't applied. On the other, it's a log and folks may complain when they see it. At least the unit won't fail now and won't show up in `systemctl --failed`. The timer triggers every 6h so maybe there would not be so much logging after all?

Reply:
I think that debugf is a bit "low" in the sense that it may not show up under (presumably) some conditions. For something as important as this, I would just print to standard output.